packages/styles/scss/_typography.scss

```diff
@@ -37,32 +37,3 @@ html[lang*="ja"] {
   --ilo-fonts-display: var(--ilo-fonts-display-jp);
   --ilo-fonts-copy: var(--ilo-fonts-copy-jp);
 }
-
```

Comment:
Nice, lets also remove other SCSS variables from `_typography` and `_typographymonorepo` 

```scss
html,
body {
  color: $color-font-base; // <--- this
  font-family: var(--ilo-fonts-copy);
  font-size: #{$type-base-size}; // <--- this
  line-height: #{$type-base-line-height};
  letter-spacing: $type-base-letter-spacing; // <--- this
  -webkit-font-smoothing: antialiased;
}
```

Reply:
I'll get to this in #1080